libraries/botbuilder/src/channelServiceRoutes.ts

```diff
@@ -402,12 +400,8 @@ export class ChannelServiceRoutes {
      * @private
      */
     private static handleError(err: any, res: WebResponse): void {
-        if (err instanceof StatusCodeError) {
-            res.send(err.message);
```

Comment:
This actually meant that errors yielded HTTP 200 status codes as you generally need to set the status code before sending a response.